website/blog/2020-02-26-cobalt.md

```diff
@@ -0,0 +1,449 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.1 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We are excited to announce the release of Metals v0.8.1. This release includes a
+large number of bug fixes and some new features.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">313</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">100</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">19</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">52</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">2</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/19?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with help from [Lunatech](https://lunatech.com) along with contributors from the
+community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/) and
+give Metals a try!
+
+- insert missing abstract members
+- new file provider
+- enable rename preview for Visual Studio Code
+- debug and run working in Emacs
+
+## Insert missing abstract members
+
+Thanks to yet again amazing work by
+[tanishiking](https://github.com/tanishiking), we now support the quick fix code
+action for implementing abstract class members. This code action is available in
+case of errors and works in the same way as the recent "Implement all members"
+completion. It will add any missing abstract members at the top of the class
+with the default implementation of ???.
+
+![action](https://user-images.githubusercontent.com/9353584/73630120-8b0bc880-4698-11ea-8370-32b1fb2f86fd.gif)
```

Comment:
Oops. Sorry again, this behavior in this screen-record is a bit different from the real (we add newlines between each member). I'll take a new one!

Reply:
Just re-took a screen-record with metals 0.8.1, can you use this one instead? :)

![action-take2](https://user-images.githubusercontent.com/9353584/75405616-81047f00-5951-11ea-9972-a12e25b1746a.gif)

Reply:
Right, I forgot about the newline. Could have made a new gif, but was plain lazy ech.